internal/app/maintenance/server.go

```diff
@@ -130,16 +141,12 @@ func (s *Server) GenerateConfiguration(ctx context.Context, in *machine.Generate
 }
 
 // GenerateClientConfiguration implements the [machine.MachineServiceServer] interface.
-func (s *Server) GenerateClientConfiguration(ctx context.Context, in *machine.GenerateClientConfigurationRequest) (*machine.GenerateClientConfigurationResponse, error) {
+func (s *Server) GenerateClientConfiguration(context.Context, *machine.GenerateClientConfigurationRequest) (*machine.GenerateClientConfigurationResponse, error) {
 	return nil, status.Error(codes.Unimplemented, "client configuration (talosconfig) can't be generated in the maintenance mode")
 }
 
 // Version implements the machine.MachineServer interface.
-func (s *Server) Version(ctx context.Context, in *emptypb.Empty) (*machine.VersionResponse, error) {
-	if err := assertPeerSideroLink(ctx); err != nil {
```

Comment:
what is the equivalent replacement for it?

Reply:
I went a bit wrong direction on this - now everything should be in order, including the replacement for this.